C/Chemfiles/bundled/patches/fileno_posix.patch

```diff
@@ -0,0 +1,24 @@
+--- a/external/netcdf/libdispatch/dfile.c
++++ b/external/netcdf/libdispatch/dfile.c
+@@ -32,7 +32,9 @@
+ #ifdef HAVE_STDIO_H
+ #include <stdio.h>
+ #endif
++#ifdef  __USE_POSIX
```

Comment:
thanks for taking the time to do this! Can I incorporate this patch directly in our fork of netcdf? https://github.com/chemfiles/netcdf-c

Reply:
Sure!